4.8/alpine/patches/ppc64le-v8patch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Using sed here may reduce the change in the future
+file="deps/v8/src/sampler.cc"
```

Comment:
Removed patch files. alpine is not supported in ppc64le for now. thanks for the comments!